### ISSUE TITLE
[Bug] Limit of 5 marks is not enforced

### ISSUE THREAD

**Issue by ProPeach** [LeMinaw/skywarehouse#13]
When rating a submission, you can enter any number, not just 1 - 5
![image](https://user-images.githubusercontent.com/27857711/53055421-8a861c00-349f-11e9-9904-6d46d92bb0c6.png)

Thanks as always!

**Comment by LeMinaw** [comment]
Yes, a value in the range 1 - 5 is enforced server-side however.
In fact, those number inputs are weird and were just placeholder for a nice "star-rating" widget. I guess I just forgot to replace it...
